source/s3_auto_ranged_put.c

```diff
@@ -322,7 +317,7 @@ static int s_s3_auto_ranged_put_request_type(struct aws_s3_request *request) {
 
 static struct aws_s3_meta_request_vtable s_s3_auto_ranged_put_vtable = {
     .update = s_s3_auto_ranged_put_update,
-    .send_request_finish = s_s3_auto_ranged_put_send_request_finish,
+    .send_request_finish = aws_s3_meta_request_send_request_finish_default,
```

Comment:
Sorry to comment on pre-existing code, but I get REALLY REALLY confused about the `vtable->send_request_finish()` vs `vtable->finished_request()` callbacks. Can we give them names that make it clear what each is doing? We can do this as a separate PR. Maybe:

request_complete() / request_utterly_complete()
request_complete_p1_maybe_retry()/ request_complete_p2_no_retry()

???

Reply:
resolving this, since it's not really related to the changes in this PR